boards/esp32-wrover-kit/board.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2019 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_esp32_esp-wrover-kit
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for esp32-wrover-kit
+ *
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#include "board.h"
+
+void board_init(void) {
```

Comment:
Style (ignore if you prefer):

```suggestion
void board_init(void)
{
```